tools/its.py

```diff
@@ -202,8 +212,9 @@ def get_chromedriver(self, headless):
             chrome_options.binary_location = os.getenv("VACCIPY_CHROME_BIN")
 
         chrome_options.headless = headless
-
-        return Chrome(self.get_chromedriver_path(), options=chrome_options)
+        
+        self._chromeInstance = Chrome(self.get_chromedriver_path(), options=chrome_options);
+        return  self._chromeInstance
```

Comment:
Semikolon und Doppelleerzeichen braucht man nicht.

```suggestion
        self._chromeInstance = Chrome(self.get_chromedriver_path(), options=chrome_options)
        return self._chromeInstance
```